test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -54,19 +54,32 @@ def test_good_manifest(self, mock_webfeatureservice):
         self.assertCountEqual(INPUT_FILE_COLLECTION.get_attribute_list('dest_path'),
                               handler.input_file_collection.get_attribute_list('dest_path')
                               )
-        self.assertEqual(len(handler.file_collection), 5)
-
+        self.assertEqual(len(handler.file_collection), 7)
```

Comment:
why the 7 hardcoding? If it's because this class is only executed for a single test, them grab the number of file_collection as args.

Reply:
I also think the test_good_manifest is doing too much stuff - maybe some function refactoring !?